internal/os/volume/api.go

```diff
@@ -19,9 +19,9 @@ func New() VolAPIImplementor {
 }
 
 func runExec(cmd string) ([]byte, error) {
-	klog.V(5).Infof("Running command: %s", cmd)
+	klog.V(4).Infof("Running command: %s", cmd)
 	out, err := exec.Command("powershell", "/c", cmd).CombinedOutput()
-	klog.V(5).Infof("Result: %s. Len: %d. Error: %v.", string(out), len(string(out)), err)
+	klog.V(4).Infof("Result: %s. Len: %d. Error: %v.", string(out), len(string(out)), err)
```

Comment:
May be V(5) here since this is going to be a lot of logs when it comes to powershell so we can do 4 for other server and os layer and 5 for just the powershell. What do you say ?

Reply:
remove the log from os